dto.c

```diff
@@ -703,23 +712,19 @@ static __always_inline  int get_numa_node(void* buf) {
 
 	switch (is_numa_aware) {
         case NA_BUFFER_CENTRIC: {
-			if (buf != NULL) {
-				int status[1] = {-1};
-
-				// get numa node of memory pointed by buf
-				if (move_pages(0, 1, &buf, NULL, status, 0) == 0) {
-					numa_node = status[0];
-				} else {
-					LOG_ERROR("move_pages call error: %d - %s", errno, strerror(errno));
-				}
+			int status[1] = {-1};
```

Comment:
Looks like some code cleanup is also present in the diff which is unrelated to the main purpose of this commit. Can you create a separate patch for the code cleanup?